### PR TITLE
Feature/ben read in nhs survey

### PR DESCRIPTION
**Pull request by benmbrew**
I'll send an email with this, explaining whats up.

### CODE REVIEW

File: explore_data.R [hunk 1]
```diff
@@ -23,160 +29,296 @@ source('global.R')
...
+    result_list[[i]] <- data_list
+
+  }
+
+  print(temp_folder)
+}
+
+length(result_list)
+
+save.image('~/Desktop/survey_temp_readin.RData')
```
Comment (joebrew) [inline explore_data.R:113]: What's going on with these lines? We shouldn't have hard-coded paths in our code base...

**Review by joebrew — CHANGES_REQUESTED** [review]

File: explore_data.R [hunk 2]
```diff
@@ -23,160 +29,296 @@ source('global.R')
...
+load('~/Desktop/survey_temp_readin.RData')
+
+survey_folders[1]
```
Comment (joebrew) [inline explore_data.R:116]: Need some comments here to know exactly what you're doing.  Why so much temp stuff?

**Comment by joebrew** [comment]
@benmbrew Before we can merge this, I have a few requests:

1. There is a "merge conflict" - this means that git can't figure out which of the changes to `global.R` it should keep and discard. This conflict needs to be resolved before we can merge. In order to resolve, get on this branch, run `git merge master` and then open the conflicting file `global.R` and manually resolve the conflicts (ie, delete things, keep things, etc.). Does that make sense? Let me know if you need help on this.

2. The "data_notes_urvey" google doc (https://docs.google.com/document/d/1QMw8sYssecFxHFNTCt9hLrkLqOv-FJ-BcITPXf28B5s/edit) should be placed into this repository (so that our code and documentation are linked / in the same place). The contents of this doc should be added to the `README.md` (and the readme should be updated more generally).

3. I've left a couple minor comments in line.

**Comment by benmbrew** [comment]
Yo, i should have said in the orignal email to ignore explore_data.R and only look at global.R

File: explore_data.R (hunk 1 above)
Comment (benmbrew) [reply explore_data.R:113]: ignore explore_data.R those temp objects were just for me to save my place.

File: explore_data.R (hunk 2 above)
Comment (benmbrew) [reply explore_data.R:116]: same here. only global.R

File: explore_data.R (hunk 1 above)
Comment (joebrew) [reply explore_data.R:113]: @benmbrew Okay.

File: explore_data.R (hunk 2 above)
Comment (joebrew) [reply explore_data.R:116]: @benmbrew Okay.

**Review by joebrew — APPROVED** [review]

**Pull request merged**